.github/workflows/release.yml

```diff
@@ -32,15 +32,25 @@ jobs:
           - target: windows-x86_64
             os: windows-latest
             bin_ext: .exe
+            cc: gcc
+            cflags: -ffunction-sections -fdata-sections
+            ldflags: -Wl,--gc-sections
           - target: linux-x86_64
             os: ubuntu-20.04
+            cc: gcc
+            cflags: -ffunction-sections -fdata-sections
+            ldflags: -Wl,--gc-sections
           - target: darwin-x86_64
             os: macos-latest
+            cc: clang
+            cflags: -ffunction-sections -fdata-sections
+            ldflags: -Wl,-dead_strip
           - target: darwin-arm64
             os: macos-latest
+            cc: clang
+            cflags: -ffunction-sections -fdata-sections
+            ldflags: -Wl,-dead_strip
```

Comment:
Compiler flag for the compilation of the v-analyzer binary are set in build.vsh script as well, depending on the build mode. It might be better to improve the flags there, that way using the build script locally gets the improvements as well.